app/javascript/components/pages/TripNew.js

```diff
@@ -44,6 +43,7 @@ export default class TripNew extends Component {
                 value={this.state.form.trip_name}
               />
             </FormGroup>
+            <br/>
```

Comment:
usually `<br/>` tags aren't used anymore because styling and spacing can be achieved through css. Also I believe for accessibility reasons, it creates noise for users who use assistive technology to navigate the page